src/corehost/cli/deps_format.cpp

```diff
@@ -350,15 +352,15 @@ bool deps_json_t::load(bool portable, const pal::string_t& deps_path, const rid_
     // If file doesn't exist, then assume parsed.
     if (!pal::file_exists(deps_path))
     {
-        trace::verbose(_X("Deps file does not exist [%s]"), deps_path.c_str());
+        trace::verbose(_X("Deps.json file [%s] does not exist"), deps_path.c_str());
```

Comment:
Let's use full names to be a little more user-friendly. How about `Could not locate dependencies list '[deps file]'. Some libraries may fail to resolve.`

In general, I think it's good to inform users what kinds of failures may occur when these messages are printed (even if they have to turn on tracing to see them). That way we get clearer bug reports and questions :)